http/src/test/resources/reference.conf

```diff
@@ -37,7 +37,7 @@ mysql {
   port = 3311
   db {
     driver = "com.mysql.cj.jdbc.Driver"
-    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/leotestdb?createDatabaseIfNotExist=true&useSSL=false&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
+    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/leotestdb?createDatabaseIfNotExist=true&&allowPublicKeyRetrieval=true&useSSL=false&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
```

Comment:
Needed now with 8.0, see https://customerportal.qlik.com/article/MySQL-JDBC-connection-Public-Key-Retrieval-is-not-allowed